### PR TITLE
Update with meeting minutes

### PR DESCRIPTION
**Pull request by seabaylea**
@swift-server/stakeholders minutes from the Security stream meeting now available for review.

### CODE REVIEW

File: meetings/security/2016-12-14/minutes.md [hunk 1]
```diff
@@ -1,19 +1,63 @@
...
-### WebEx and Agenda/Minutes doc:
-* WebEx:     https://meetings.webex.com/collabs/#/meetings/detail?uuid=M6T97FS5EMMK5RKXHFH0P5V2MR-1T5J&rnd=352264.78890
-* Agenda:    https://docs.google.com/document/d/168o2pYw4IBwv93bd61-_Y5y91GLxFpHxQ29gmExx_zs/edit?usp=sharing
+### Attendees:
+* Chris Bailey
+* David Sperling
+* Gelareh Taban
+* Alfredo Delli Bovi
+* Bill Abt
+* Alex Blewitt
```
Comment (alblue) [inline meetings/security/2016-12-14/minutes.md:14]: Worth sorting alphabetically, like last time?

File: meetings/security/2016-12-14/minutes.md [hunk 2]
```diff
@@ -1,19 +1,63 @@
...
+# Minutes:
 
-### Initial Agenda:
-* Current discussion status roundup
-* Review of BlueSSLService, a layer on top of OpenSSL and macOS Secure Transport libraries
+_Scope of the Security APIs:_  
+* The scope of what's eventually intended is the full set of APIs, spanning across secure transport (SSL/TLS), crypto, and keychain/certificate management, etc
+* The aim is also to provide a consistent set of APIs across both clients and servers, although not all APIs may be available in every case
+
+ which crypto depends on what’s available (and whether there’s requirements above/beyond what we get from the macOS and/or OpenSSL/LibreSSL libs).
```
Comment (alblue) [inline meetings/security/2016-12-14/minutes.md]: there's -> there are

Maybe also capitalise 'W' in 'Which' at the start?

File: meetings/security/2016-12-14/minutes.md [hunk 3]
```diff
@@ -1,19 +1,63 @@
...
+
+_Review of approach taken in BlueSSLService:_  
+* BlueSSLService covers the "secure transport" part of the APIs. There's also a BlueCryptor library that covers crypto.
+* It provides a service that adds onto an existing socket library conforming to a protocol. The only implementation of that is currently BlueSocket, but it could be easily modified to work with other socket implementations.
+  * 6 protocol methods need to be implemented in the socket library.
+  * A method in the SSL service to let you override the connection verification process to allow for specialized connection verification on an application basis. 
+* The initial implementation of BlueSSLService uses OpenSSL on all platforms (including macOS), but was then modified to use SecureTransport on macOS.
+  * The APIs are very different between OpenSSL and SecureTransport, and there was some struggle with the SecureTransport documentation.
+  * OpenSSL provides much more flexibility for configuration. macOS/SecureTransport is limited to using pkcs#12, although there are internal APIs for dealing with other formats that are not exposed.
```
Comment (alblue) [inline meetings/security/2016-12-14/minutes.md]: Worth mentioning any particular formats that aren't exposed here?

File: meetings/security/2016-12-14/minutes.md [hunk 4]
```diff
@@ -1,19 +1,63 @@
...
+
+_Use of BlueSSLService and BlueCryptor as a prototype implementation:_  
+* Should we use BlueSSLService and BlueCryptor as a prototype implementation, and use that to work on the API surface?
+* This seems like a reasonable approach
 
-If you'd like to add anything to the agenda ahead of time, please update the agenda document.
+### Next Steps:
+* Discuss the implemented, but not exposed, function in SecureTransport and CommonCrypto with Luke Hiesterman
+* Carry out more investigation on the need (if any) for common certificate/keychain management
+* Start to build an outline proposal for the HTTP parsing library for use as a Swift Evolution "pitch"
```
Comment (alblue) [inline meetings/security/2016-12-14/minutes.md]: Isn't the HTTP parsing library being addressed by a different working group? If not, can you be more specific here e.g. whether this is looking at a lower layer socket library or the ability to add https?

File: meetings/security/2016-12-14/minutes.md (hunk 2 above)
Comment (seabaylea) [reply meetings/security/2016-12-14/minutes.md]: Done

File: meetings/security/2016-12-14/minutes.md (hunk 3 above)
Comment (seabaylea) [reply meetings/security/2016-12-14/minutes.md]: Added `(such as PEM)`, which then leads in better to the next line.

File: meetings/security/2016-12-14/minutes.md (hunk 4 above)
Comment (seabaylea) [reply meetings/security/2016-12-14/minutes.md]: Ahh, a classic cut-and-paste failure! Should be `Security libraries`.

File: meetings/security/2016-12-14/minutes.md (hunk 1 above)
Comment (seabaylea) [reply meetings/security/2016-12-14/minutes.md:14]: I'm not sure there's much benefit to listing the attendees alphabetically, so I'll leave as-is (we've never consciously listed alphabetically in the past!).

**Comment by seabaylea** [comment]
Thanks for the review @alblue . I've made updates based on it.

**Review by adellibovi — APPROVED** [review]

**Comment by seabaylea** [comment]
Merging.

**Pull request merged**